### ISSUE TITLE
Feature request? 

### ISSUE THREAD

**Issue by fwanky** [naelstrof/maim#177]

**Comment by naelstrof** [comment]
Yeah! All you have to do is take a full screenshot and pipe it into a full screen image viewer like feh. From there you can launch a maim -s to take a crop of the still.

I think there's an example in the readme about it.

**Comment by foxpy** [comment]
I think the conversation has moved to #193, this issue should be closed now.
